### DISCUSSION TITLE
Cannot setup autoboot, hidden boot option works

### DISCUSSION THREAD

**Discussion by alaconte** [CloverHackyColor/CloverBootloader#622] [Q&A]
I've been trying to set a default boot option but I cannot get anything to work, so I'd appreciate some help. I am able to boot into my target os (pfSense) by hitting f3 to show hidden entries and selecting it from there, but ideally I don't want to have to do this every time.

I have attached the extra info I get by hitting space over the boot option that works, as well as my config.plist and preboot.log.
[preboot.log](https://github.com/CloverHackyColor/CloverBootloader/files/12264412/preboot.log)
[config.plist.txt](https://github.com/CloverHackyColor/CloverBootloader/files/12264414/config.plist.txt)
![IMG_20230804_141904](https://github.com/CloverHackyColor/CloverBootloader/assets/77861389/85dfc31b-05f2-44c0-a32d-58870e96038c)

**Comment by siliconbeaver** [comment]
I had a similar issue. CloverISO-5070. got fixed.

DefaultVolume and GUID.
my problem was that initial GUID was wrong (typo).
I read debug.log carefully. then right copied the GUID. re-start. it boot into OS when timeout.

**Comment by alaconte** [comment]
I ended up switching to DUET-REFIND. My issue was definitely not having the wrong guid as I was able to check with both the log and after booting into my target OS. Don't need help anymore though so I'm going to mark this as closed, but if anyone is having this issue I'd recommend switching to a different tool.

**Comment by siliconbeaver** [comment]
For your DUET-REFIND method, what USB drive size required? 128MB? 1GB? winraid didn't say that.
